backend/src/agents/config/tasks.yaml

```
@@ -0,0 +1,105 @@
+                      
+                }
+            ]
+        }
+    }
+  input_variables:
+    - user_request
+    - agents_list
+    - examples_list
+
+execution_task:
+    agent: executor_agent
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)